components/main-with-sidebar/main-with-sidebar--sidebar-menu.html

```diff
@@ -0,0 +1,9 @@
+<div class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-left">
```

Comment:
**Praise:** I like that you created this file to show a realistic demo. I'm happy to keep it (it's super useful imo), however I do know that it's not necessarily a variant, which is what we've been using these page views for. @craigcook thoughts?

Reply:
Given that we don't currently have a replacement for the old demo pages, I think we can still include useful examples in Fractal, but I agree they probably don't count as "variants". 

But we can add a template with a different filename (not a `--` variant) in the same folder and Fractal will treat it as another "component" in the UI rather than a variant. I did that with an example form: https://demo1--mozilla-protocol.netlify.app/components/detail/example-form.html

Reply:
@craigcook let me know what you think of the general structure here. I wasn't able to pull the example sidebar menu out without grouping into nested folders. Is this an improvement? Or do we want to fallback to the pseudo-variant sidebar menu example for now?

<img width="220" alt="menu-nesting" src="https://user-images.githubusercontent.com/19650432/173051661-b437f703-89dc-4f92-bf68-a6a35060d4ef.png">
<img width="236" alt="folder-nesting" src="https://user-images.githubusercontent.com/19650432/173051663-efc5e3ce-a1ba-474e-b783-31a97f3476dc.png">

Reply:
LGTM. Numbering filenames and folders is how Fractal explicitly tells you to order things (Drizzle would let us define the order in a config).